server/proxy/synchronize.go

```diff
@@ -24,8 +23,7 @@ func (p *Proxy) SynchronizeInstalledApps() error {
 	ctx, cancel := context.WithTimeout(context.Background(), config.RequestTimeout)
 	defer cancel()
 
-	mm := p.conf.MattermostAPI()
-	r := incoming.NewRequest(p.conf, utils.NewPluginLogger(mm), p.sessionService).WithCtx(ctx)
```

Comment:
Should we pass a `Logger` to `SynchronizeInstalledApps`?

Reply:
1/5 it's fine as is, since it creates a new request with its own logger and uses it. The old explicit log initialization is already gone.